vault/core.go

```diff
@@ -725,6 +725,14 @@ func NewCore(conf *CoreConfig) (*Core, error) {
 		return nil, errwrap.Wrapf("barrier setup failed: {{err}}", err)
 	}
 
+	// We create the funcs here, then populate the given config with it so that
```

Comment:
Why was this done? Were we hitting a panic due to reload funcs not being initialized?

Reply:
Correct.  NewTestCluster relies on the map being non-nil.